Tests/DirectoryTests.swift

```diff
@@ -190,11 +189,11 @@ class DirectoryTests: XCTestCase {
         let textExpectation = expectation(description: "testRead")
         
         let textA = "This is a test read A"
-        try? textA.data(using: .utf8)?.write(to: cacheDirectory.appendingPathComponent("testReadA", isDirectory: false))
+        ((try? textA.data(using: .utf8)?.write(to: cacheDirectory.appendingPathComponent("testReadA", isDirectory: false))) as ()??)
```

Comment:
Let's see if we can fix these double optionals (I assume the migrator did this).

Reply:
It was the migrator.

Reply:
fixed